Refactor Copyable component to improve alignment of icons and structure of JSX elements

Code-Parth opened the pull request:
## Description

Refactored the `Copyable` component to improve the alignment of icons and the overall structure of JSX elements. Adjusted spacing and layout to ensure better visual consistency.  

## Type of change  

-   [ ] Bug fix  
-   [ ] New feature  
-   [ ] Protocol integration  
-   [ ] Documentation update  
-   [x] Other (please describe): `bootstrap CSS issue`

## Screenshots  


https://github.com/user-attachments/assets/1cfaa9d2-fb57-4f5b-afcf-db0730982d11




## Checklist  

-   [x] My code follows the project's style guidelines  
-   [x] I have added tests that prove my fix/feature works  
-   [x] All tests pass locally and in CI  
-   [ ] I have updated documentation as needed  
-   [x] CI/CD checks pass  
-   [ ] I have included screenshots for protocol screens (if applicable)  
-   [ ] For security-related features, I have included links to related information

ngundotra commented:
![Screenshot 2025-03-27 at 5 14 53 PM](https://github.com/user-attachments/assets/d46202dc-4c54-4c34-a253-a963606daaf3)

Hey this seems to pretty drastically change alignment of copyable elements. Do you mind fixing this?

ngundotra reviewed (changes requested):
This pretty drastically changes UI elements outside the transaction page.

See the home page for an example

Code-Parth commented:
Thank you for your feedback! I appreciate your review.

The primary goal of this refactor was to improve icon alignment and ensure better visual consistency within the Copyable component.

I changed the alignment from `align-text-top` to `align-middle` to enhance the positioning of the icons.

Could you please clarify which specific misalignments you are noticing and on which page?

I did not see the response on the homepage for the mainnet. However, now that I have switched to Devnet, I can see this issue.

Code-Parth commented on app/components/common/Copyable.tsx:
```diff
@@ -53,29 +53,29 @@ export function Copyable({
         return (
-            <>
-                <span className="font-size-tiny me-2">
+            <div className="d-flex align-items-center">
```
Here, I add `justify-content-end` to fix that issue

Code-Parth commented:
Also, I checked some of the pages for the copy button, and that is perfectly aligned, not mismatched
